common/libs/nodes/rpc/index.js

```diff
@@ -71,13 +63,19 @@ export default class RpcNode extends BaseNode {
 
   async generateTransaction(
```

Comment:
Yeah, this got a lot bigger than I initially intended. What's our current pattern for doing rpc calls in separate libraries? Do you pass the node into each function, or have a class with a constructor that takes it in?